src/ucp/core/ucp_request.inl

```diff
@@ -50,17 +50,17 @@
 
 
 /* defined as a macro to print the call site */
-#define ucp_request_get(_worker) \
-    ({ \
+static inline ucp_request_t* ucp_request_get(ucp_worker_h _worker) \
```

Comment:
- `USC_F_ALWAYS_INLINE`
- pls remove ending `\`s, no need in func

Reply:
and move function below defines

Reply:
also func names do not start with `_`